### PR TITLE
feat(payment): update payment success page to new designs

### PR DESCRIPTION
**Pull request by vbudhram**
## Because

- We need to update these to match new figma designs

## This pull request

- Updates designs from https://www.figma.com/file/cOIAuHIIbrPbAlAIcpejNM/%F0%9F%8E%A1MVP-Subscription-Optimization-Flow?node-id=400%3A2098

## Issue that this pull request solves

Closes: https://github.com/mozilla/fxa/issues/9365

## Checklist

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

**Comment by biancadanforth** [comment]
@vbudhram Hello! Would you mind [adding a comment in `main.ftl` explaining what `productName` is](https://github.com/mozilla/fxa/pull/10127#discussion_r690530258) for the message string with id `"fxa-account-signup-error"`? I neglected to do that in my PR that was merged earlier.

**Comment by meandavejustice** [comment]
The `yarn build` command is failling in ci:

```
TypeScript error in /home/circleci/project/packages/fxa-payments-server/src/routes/Product/index.tsx(208,10):
Property 'accountExists' is missing in type '{ plan: Plan; customer: Customer; profile: Profile; isMobile: boolean; }' but required in type 'SubscriptionSuccessProps'.  TS2741

    206 |     if (alreadySubscribedToSelectedPlan) {
    207 |       return (
  > 208 |         <SubscriptionSuccess
        |          ^
    209 |           {...{
    210 |             plan: selectedPlan,
    211 |             customer: customer.result,
```

File: packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx [hunk 1]
```diff
@@ -65,19 +69,33 @@ export const PaymentConfirmation = ({
...
           </Localized>
           <Localized
-            id="payment-confirmation-thanks-subheading"
-            vars={{ email, product_name }}
+            id="payment-confirmation-thanks-subheading-2"
+            vars={{ email }}
           >
-            <p>{`A confirmation email has been sent to ${email} with details on how to get started with ${product_name}.`}</p>
+            <p>{`You'll receive an email at ${email} with instructions for setting up your account as well as  your payment details.`}</p>
           </Localized>
```
Comment (biancadanforth) [inline packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx:99]: We probably need to differentially show these messages based on `accountExists` too. Right now I see the passwordless strings when I sign up with an existing FxA account locally.

**Review by meandavejustice — CHANGES_REQUESTED** [review]
This looks pretty good, I noticed a couple little style tweaks to get this to match the figma more precisely.

1. It looks like font-weight needs to be set on the `.subtitle` element.
2. spacing between `.subtitle` and the header containing `.checkmark` icon should be closer to 32px
3. spacing between legal links at the bottom does not match the initial state of the checkout page.

image

image

**Review by biancadanforth — COMMENTED** [review]
Hey Vijay -- I know this is still a WIP. I was just passing through and thought I'd see how things were going. :-)

**Review by meandavejustice — APPROVED** [review]
LGTM, thanks for picking up this task last minute @vbudhram, we appreciate it :)

**Pull request merged**